pages/recs/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { AUTH_TOKEN } from "~/config/constants";
+import { fetchUnifiedDocFeed } from "~/config/fetch";
+import { isNullOrUndefined } from "~/config/utils/nullchecks";
+import HubPage from "~/components/Hubs/HubPage";
+import nookies from "nookies";
+import { getSelectedUrlFilters } from "~/components/UnifiedDocFeed/utils/getSelectedUrlFilters";
+import { getFetchDefaults } from "~/components/UnifiedDocFeed/utils/getFetchDefaults";
+import { NextPage } from "next";
+import HeadComponent from "~/components/Head";
+
+const Index: NextPage = (props) => {
+  return (
+    <div>
+      <HeadComponent
+        title={"ResearchHub | For You"}
+        description={"Discuss and Discover the latest research on ResearchHub"}
+      />
+      <HubPage isForYouFeed={true} {...props} />
+    </div>
+  );
+};
+
+Index.getInitialProps = async (ctx) => {
+  const { query } = ctx;
+  const cookies = nookies.get(ctx);
+  const authToken = cookies[AUTH_TOKEN];
+  const defaultProps = getFetchDefaults({ query, authToken });
+
+  if (process.browser) {
+    return defaultProps;
+  }
+
+  try {
+    const selectedFilters = getSelectedUrlFilters({ query, pathname: "/" });
+    const initialFeed = await fetchUnifiedDocFeed(
```

Comment:
Can we remove this for now? I think it is causing the serverless function to crash